### PR TITLE
add new ApiCaller to delegate api calls

### PR DESCRIPTION
**Pull request by sensorario**
- also introduced Makefile to perform tasks
 - add coverage (in Makefile)
 - reached 100% of coverage ^_^

### CODE REVIEW

File: src/ApiCaller.php [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+            . '?q=' .  $from . '_' . $to
+            . '&compact=ultra' ;
+
+        if ($handle = @fopen($url, 'r')) {
+            $this->response = fgets($handle, 4096);
+            fclose($handle);
+        }
+    }
+
+    public function isLastCallEmpty()
```
Comment (AlessandroMinoccheri) [inline src/ApiCaller.php]: It could be better to add return type?

File: src/ApiCaller.php [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    {
+        return $this->response == null;
+    }
+
+    public function getLastResponse()
```
Comment (AlessandroMinoccheri) [inline src/ApiCaller.php]: Same as above.

File: Makefile [hunk 3]
```diff
@@ -0,0 +1,16 @@
+.MINO: agile
```
Comment (AlessandroMinoccheri) [inline Makefile:1]: Can you please add into README file how to use Makefile please?

**Review by AlessandroMinoccheri — CHANGES_REQUESTED** [review]

File: Makefile (hunk 3 above)
Comment (sensorario) [reply Makefile:1]: done!

**Review by AlessandroMinoccheri — APPROVED** [review]
LGTM 👍

**Pull request merged**